lib/agent/plugins/control-panel/interval/index.js

```diff
@@ -83,7 +92,10 @@ var unload = function(err) {
 
   hooks.remove('woken', request);
```

Comment:
It's a different thing. The plugin's main controller waits for a 'woken' event from the **push adapter** (not the hooks module). When that occurs, it calls `interval.check()`. The 'woken' event from hooks is an old thing -- I think I just forgot to remove it. Actually, I think `hooks.trigger('woken')` doesn't happen anywhere anymore.